src/layer/place.js

```diff
@@ -451,4 +457,14 @@ export const legendEntries = [
     layers: populatedPlaceLayers,
     filter: ["==", ["get", "capital"], 4],
   },
+  {
+    description: "Division capital",
+    layers: populatedPlaceLayers,
+    filter: ["==", ["get", "capital"], 5],
+  },
+  {
+    description: "County seat or local capital",
+    layers: populatedPlaceLayers,
+    filter: ["==", ["get", "capital"], 6],
+  },
```

Comment:
Legend labels for `capital=*` are gonna depend heavily on the country. Not sure what the best approach is here.

Reply:
New York City is a "Division" capital 🤔